pkg/informer/informer.go

```diff
@@ -394,16 +394,16 @@ func (d *GenericDiscoveringDynamicSharedInformerFactory[Informer, Lister, Generi
 
 // Informer returns an informer for the given resource-type if it exists, whether it is
 // known by this informer factory, and whether it is synced.
-func (d *GenericDiscoveringDynamicSharedInformerFactory[Informer, Lister, GenericInformer]) Informer(gvr schema.GroupVersionResource) (informer Informer, known, synced bool) {
+func (d *GenericDiscoveringDynamicSharedInformerFactory[Informer, Lister, GenericInformer]) Informer(gvr schema.GroupVersionResource) (informer Informer, known bool) {
```

Comment:
known is actually a strange name and https://github.com/kcp-dev/kcp/pull/2452/files#r1064592734 is a strange logic. Too much about state transitions, which is fragile.

Reply:
nvmd. I thought this func would start an informer.

Reply:
Still fixed as part of the refactoring mentioned here: https://github.com/kcp-dev/kcp/pull/2452#discussion_r1064848346